vue/src/store/generated/mande-chain/mandechain.voting/module/types/voting/tx.ts

```diff
@@ -29,7 +29,7 @@ export const MsgCreateVote = {
       writer.uint32(18).string(message.receiver);
     }
     if (message.count !== 0) {
-      writer.uint32(24).int32(message.count);
+      writer.uint32(24).int64(message.count);
```

Comment:
Tested the changes on UI too?

Reply:
Yes. I tested the changes both on UI and cli. It is working fine.

On UI we have a limit for 1000 so could only test with that value.

Reply:
Can you please remove the 1000 limit condition from the code and then test. Just to be sure
For ref - https://github.com/mande-labs/mande-chain/pull/23/files#diff-3b8055e901d876b3fdae9172632d08a64592a9c9474e8ef312fe6445cebae3aaR291

Reply:
Tested that. The first TX for 5000mand goes through but the second one gets internal server error.

Reply:
I think it's better if I don't include UI changes in the PR. Better to do it separately.